### ISSUE TITLE
I cannot search any of my installed apps

### ISSUE THREAD

**Issue by im-AMS** [ChrisTitusTech/win10script#16]

**Comment by gfelipe099** [comment]
That service and functionality gets disabled amog other stuff.

**Comment by im-AMS** [comment]
can u make a guess which one is exactly causing it... I have tried most of it... I have enabled indexing all those

**Comment by amithiel** [comment]
> can u make a guess which one is exactly causing it... I have tried most of it... I have enabled indexing all those

I wish to know that too... i can search the old apps that were already indexed. After i ran the script, every new app isn't searchable.
What a pain in the ass.  tried to restore de indexing service, no joy. Windows troublshooter says "search" isnt working, and fixes it, but if you run the troubleshooter again... it remains unfixed.
don't know what to restore anymore

**Comment by SleepyBandit** [comment]
I was able to resolve this for my PC. The search essentially would not find any new apps installed after running the debloater script. I had tweaked the script originally to fit my own needs, and ran reactivated a few things but it did not seem to fix it. However doing the following worked for me.
https://www.tenforums.com/general-support/9263-were-getting-search-ready-error-windows-cortana-search-2.html#post302218

Steps were
1. Disable Search Service
2. Stop search service
3. Move Data folder from C:\ProgramData\Microsoft\Search location
4. Restart

After restarting even before I reactivated the search service it seemed to have fixed it. I went back in and enabled the search service and now I can find my apps. What a relief!
